src/reflect/scala/reflect/io/ZipArchive.scala

```diff
@@ -109,10 +109,10 @@ abstract class ZipArchive(override val file: JFile, release: Option[String]) ext
     }
   }
 
-  private def ensureDir(dirs: java.util.Map[String, DirEntry], path: String, zipEntry: ZipEntry): DirEntry = {
+  private def ensureDir(dirs: java.util.Map[String, DirEntry], path: String): DirEntry = {
```

Comment:
When I touched it on 2.13, I moved the method to local of getDir.

Reply:
Thanks, that'll help my when merging next time.